test/integration/dns_test.go

```diff
@@ -187,20 +187,29 @@ func TestDNS(t *testing.T) {
 			expect:          []*net.IP{&headlessIP},
 		},
 		{ // specific port of a headless service
-			dnsQuestionName: "unknown-port-2345.e1.headless.default.svc.cluster.local.",
+			dnsQuestionName: "_http._tcp.headless.default.svc.cluster.local.",
```

Comment:
Those are tested in the extended DNS test and the test-cmd DNS test.

On Thu, Aug 4, 2016 at 11:02 PM, Solly Ross <notifications@github.com>
wrote:

> In test/integration/dns_test.go
> <https://github.com/openshift/origin/pull/9972#discussion_r73636324>:
>
> > @@ -187,20 +187,29 @@ func TestDNS(t *testing.T) {
> >  			expect:          []*net.IP{&headlessIP},
> >  		},
> >  		{ // specific port of a headless service
> > -			dnsQuestionName: "unknown-port-2345.e1.headless.default.svc.cluster.local.",
> > +			dnsQuestionName: "_http._tcp.headless.default.svc.cluster.local.",
>
> It looks like none of these exercise the _<protocol>.<svc>.<ns>.svc case
> (wildcard port match), so there should probably be such a test here.
> Furthermore, it looks like none of these test the .endpoints queries, and
> that seems like it would be valuable to test here as well.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9972/files/cc585097d4706f1d49fb9a451760f38e9863a794#r73636324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pymEzpOhGu0rsHQnD2TGkhSFO4LAks5qcqeugaJpZM4JRYA_>
> .
>